README.md

```diff
@@ -25,57 +25,51 @@
+**Meilisearch** is an open-source search engine. [Learn more about Meilisearch.](https://github.com/meilisearch/meilisearch)
 
 ## Table of Contents <!-- omit in toc -->
 
 - [📖 Documentation](#-documentation)
 - [🔧 Installation](#-installation)
-- [🎬 Getting Started](#-getting-started)
-- [🤖 Compatibility with Meilisearch](#-compatibility-with-meilisearch)
-- [💡 Learn More](#-learn-more)
-- [⚙️ Development Workflow and Contributing](#️-development-workflow-and-contributing)
+- [🎬 Getting started](#-getting-started)
+- [🤖 Compatibility](#-compatibility-with-meilisearch)
+- [💡 Learn more](#-learn-more)
+- [⚙️ Contributing](#️-contributing)
 - [📜 API Resources](#-api-resources)
 
 ## 📖 Documentation
 
-See our [Documentation](https://docs.meilisearch.com/learn/tutorials/getting_started.html) or our [API References](https://docs.meilisearch.com/reference/api/).
+This readme contains all the documentation you need to start using this Meilisearch SDK.
+
+For general information on how to use Meilisearch—such as our API reference, tutorials, guides, and in-depth articles—refer to our [main documentation website](https://docs.meilisearch.com/).
 
 ## 🔧 Installation
 
-We only guarantee that the package works with `node` >= 12 and `node` <= 16.
+We recommend installing `meilisearch-js` in your project with your package manager of choice.
 
-With `npm`:
+If you use `npm`:
 
 ```sh
 npm install meilisearch
 ```
 
-With `yarn`:
+If you prefer `yarn`:
 
 ```sh
 yarn add meilisearch
 ```
 
-### 🏃‍♀️ Run Meilisearch <!-- omit in toc -->
-
-There are many easy ways to [download and run a Meilisearch instance](https://docs.meilisearch.com/reference/features/installation.html#download-and-launch).
+`meilisearch-js` officially supports `node` versions >= 12 and <= 16.
 
-For example, using the `curl` command in [your Terminal](https://itconnect.uw.edu/learn/workshops/online-tutorials/web-publishing/what-is-a-terminal/):
+Instead of using a package manager, you may also import the library directly into your [HTML via a CDN](#include-script-tag).
 
-```bash
-# Install Meilisearch
-curl -L https://install.meilisearch.com | sh
-
-# Launch Meilisearch
-./meilisearch --master-key=masterKey
-```
+### Run Meilisearch <!-- omit in toc -->
 
-NB: you can also download Meilisearch from **Homebrew** or **APT** or even run it using **Docker**.
+To use one our SDKS, you must first have a running Meilisearch instance. Consult our documentation for [instructions on how to download and launch Meilisearch](https://docs.meilisearch.com/reference/features/installation.html#download-and-launch).
 
 ### Import <!-- omit in toc -->
 
-Depending on the environment in which you are using Meilisearch, imports may differ.
+After installing `meilisearch-js`, you must import it into your application. There are many ways of doing that depending on your development environment.
 
 #### Import Syntax <!-- omit in toc -->
```

Comment:
@bidoubiwa, what do you think about changing this heading from "Import syntax" to something like ES6 modules"?

Reply:
Based on our call earlier:

```suggestion
#### `import` syntax <!-- omit in toc -->
```

Reply:
Good idea !